### PR TITLE
feat(combobox): provide new error style

### PR DESCRIPTION
**Pull request by boiarchukk**
Create UI for combobox error-style
## Description

ComboBox provide it with error message output for invalid selections.

## Motivation and Context
UI changes

## Screenshots:
**After**:
Снимок экрана 2021-03-05 в 18 10 29

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/[sandbox]/sandbox.mock.ts [hunk 1]
```diff
@@ -395,77 +393,3 @@ export const DraggableOptions = [
...
-  config: {
-    isStickyHeader: true,
-
-    rows: {
-      isDraggable: false,
-      selectable: "multiple"
-    },
-
-    cols: {
-      isResizable: true,
```
Comment (futurethang) [inline web-components/src/[sandbox]/sandbox.mock.ts:409]: @boiarchukk Is all this being removed by accident?

File: web-components/src/[sandbox]/sandbox.ts [hunk 2]
```diff
@@ -122,14 +122,6 @@ export class Sandbox extends LitElement {
...
         ${this.themeToggle()}
         <elix-list-explorer class="explorer">
-
-          <div class="container" aria-label="md-table-advanced">
-            <h2>md-table-advanced</h2>
-            <sass-stats component="table-advanced">
-              ${tableAdvancedTemplate}
-            </sass-stats>
-          </div>
-
```
Comment (futurethang) [inline web-components/src/[sandbox]/sandbox.ts:132]: Here too, seems unrelated to ComboBox, but not clear if it is cleanup or accidental removal

File: web-components/src/[sandbox]/sandbox.ts [hunk 3]
```diff
@@ -415,10 +407,10 @@ export class Sandbox extends LitElement {
             </sass-stats>
           </div>
 
-          <div class="container" aria-label="md-table">
-            <h2>md-table</h2>
-            <sass-stats component="table">
-              ${tableTemplate}
```
Comment (futurethang) [inline web-components/src/[sandbox]/sandbox.ts:421]: Here too, seems unrelated to ComboBox, but not clear if it is cleanup or accidental removal

**Review by futurethang — CHANGES_REQUESTED** [review]

File: web-components/src/[sandbox]/sandbox.ts (hunk 2 above)
Comment (elenagiere) [reply web-components/src/[sandbox]/sandbox.ts:132]: Probably just put in in alphabetical order here. I agree that I was confusing so many changes were being made for table-advanced

Comment (futurethang) [reply web-components/src/[sandbox]/sandbox.ts:132]: @elenagiere @boiarchukk Aha, yes that's it!

File: web-components/src/[sandbox]/sandbox.mock.ts (hunk 1 above)
Comment (boiarchukk) [reply web-components/src/[sandbox]/sandbox.mock.ts:409]: @futurethang It was removed with purpose to cleanup mock data duplication for Advance Table component.

File: web-components/src/[sandbox]/sandbox.ts (hunk 3 above)
Comment (boiarchukk) [reply web-components/src/[sandbox]/sandbox.ts]: @yananym fixed

**Review by yananym — APPROVED** [review]

**Review by futurethang — APPROVED** [review]

**Pull request merged**